smartcosmos-framework/src/main/java/net/smartcosmos/security/user/SmartCosmosCachedUser.java

```diff
@@ -14,9 +14,8 @@
  */
 public class SmartCosmosCachedUser extends SmartCosmosUser {
 
-    // TODO Caching time is hardcoded at 5 minutes, should change this.
-    private final Date cachedDate = new Date(
-            System.currentTimeMillis() + TimeUnit.MINUTES.toMillis(5));
+    // Caching time is set in auth server config smartcosmos.security.cachedUserKeepAliveSecs
```

Comment:
The config path actually is `smartcosmos.security.resource.cachedUserKeepAliveSecs`, see [SecurityResourceProperties.java#L15](https://github.com/SMARTRACTECHNOLOGY/smartcosmos-framework/blob/44d0b18ee5e1409a84cc55622e9b39600c45a498/smartcosmos-framework/src/main/java/net/smartcosmos/security/SecurityResourceProperties.java#L15)